README.md

```diff
@@ -1,21 +1,66 @@
-Nomad IIS Driver
-==========
+# Nomad IIS Driver
 
 A driver plugin for nomad to orchestrate windows IIS website tasks.
 
 A "Website" is a combination of an application pool and a site (app, vdir, etc.).<br/>
 Each allocation will create an application pool and site with the name being the allocation ID (guid).
 
-Requirements
--------------------
+This driver is heavily tested on Windows 2016+ and does not guarantee compatibility with older versions of Windows/IIS. IIS version and Windows versions are locked together and can be seen [here](https://en.wikipedia.org/wiki/Internet_Information_Services). Each feature/config has a minimum IIS version associated so that one can dance around them to have nomad and the IIS driver work on an older machine. The easiest way to utilize unique features for a given IIS version should be to use the application pool and site config imports provided by the task config.
+
+---
+## Configuration
+### **Driver Config**
+
+| Option | Type | Required | Default | Description |
+| :---: | :---: | :---: | :---: | :--- |
+| **enabled** | bool | no | true | Enable/Disable task driver. |
+| **stats_interval** | string | no | 1s | Interval for collecting `TaskStats` |
```

Comment:
I know the default is `1s`, but should this be `5s` for our project, since we are [`overriding`](https://github.com/Roblox/nomad-driver-iis/blob/master/vagrant/win_client.hcl#L28) the value

Reply:
No, that is an example config being only used by vagrant. Default is still 1s.